gamemodes/terrortown/gamemode/server/sv_player_ext.lua

```diff
@@ -190,17 +190,17 @@ end
 -- @param string id
 -- @realm server
 function plymeta:AddEquipmentItem(id)
-	if not self:HasEquipmentItem(id) then
-		self.equipmentItems = self.equipmentItems or {}
-		self.equipmentItems[#self.equipmentItems + 1] = id
+	local item = items.GetStored(id)
 
-		local item = items.GetStored(id)
-		if item and isfunction(item.Equip) then
-			item:Equip(self)
-		end
+	if not item then return end
```

Comment:
I tested it with the new armor, the default TTT items and "a second chance". Without this check, buying an item that is not installed on the server, throws an error. This was never notived earlier, since IsItem prevented this.